deployments/platform-dev/docker-compose.yml

```diff
@@ -82,8 +82,25 @@ services:
       - MYSQL_PASSWORD
       - MYSQL_USER
       - MYSQL_DATABASE=pathwar
+    expose:
+      - 3306
```

Comment:
```suggestion
```

You should not expose this port, you should only use the same networks for containers that are related, but that's strange you need to do it, this container is already used by other containers (see above)

Reply:
Indeed, the port is already exposed in Dockerfile, my bad